### ISSUE TITLE
if it can use by mt7916 wifi chip?

### ISSUE THREAD

**Issue by SeanPcWoo** [rdkcentral/RdkEasyMeshController#8]
Hi, I am do a project in RTOS. In my project, wifi chipset is MT7916, CPU is a arm64 chipset.
The MT7916 can work normally in my Porject with AP mode. 

Now I want use Mt7916 to work as a WiFi Mesh Controller. However, Mediatek can support only part code of the WiFi Mesh, some code in 'mapd'(one of Mediatek  wifi mesh apps) is not open source. So I can not use the offical code in My RTOS.

I find this open source Project, and code is very beautiful, I like it very much. And Could U tell me if this project can help me do WiFi Mesh Controller with MT7916 chipset in RTOS.

If the answer is yes, I will spend time to learn and port these code to My RTOS.

Thanks!

**Comment by cpokuru** [comment]
@SeanPcWoo ,Are you using RDK-B,can you give little more background.

**Comment by SeanPcWoo** [comment]
No, My hardware platform is RK3568, wifi chipset is MT7916. My OS is a RTOS. I have port the MT7916 driver to RTOS, it works well in Ap mode. Now I want  port this repository to My RTOS and help me do WiFi Mesh Controller. I have no idea that this repository can do the job.

**Comment by cpokuru** [comment]
Basically this is designed to serve as an easymesh controller.airties(https://airties.com/) is the developer for this repo and contributed to RDK-B community.

**Comment by torrentius** [comment]
EM Controller is independent from the wifi SoC you are using, the code could be portable to the RK3568 if you have access to the tool chain, I would try to build the Meshcoms project first where you have build instructions in the README, if it works you will be able to build the controller. We use Yoco what it simplifies the build dependencies, if you plan to use regular makefiles, it will be a bit harder, good luck!

**Comment by SeanPcWoo** [comment]
@torrentius Thanks! I have tried to build in my project. Now, I found 2 problems: 
1. My system not support rbus;
2. I can not found some ccsp api and header files.

If i solve these 2 problem. EM controller may be work in my Project.

**Comment by cpokuru** [comment]
@SeanPcWoo ,rbus is RDK-B specific,but it is generic (https://github.com/rdkcentral/rbus) and ccsp api's is part of RDK-B.but generic api's are open.you can refer https://code.rdkcentral.com/r/admin/repos
